finished/src/main/java/com/android/example/wear/ongoingactivity/presentation/OngoingActivityExampleApp.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android.example.wear.ongoingactivity.presentation
+
+import android.Manifest
+import android.os.Build
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.android.example.wear.ongoingactivity.MainViewModel
+import com.android.example.wear.ongoingactivity.MainViewModelFactory
+import com.android.example.wear.ongoingactivity.data.WalkingWorkoutsRepository
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import com.google.accompanist.permissions.PermissionStatus
+import com.google.accompanist.permissions.rememberPermissionState
+
+@OptIn(ExperimentalPermissionsApi::class)
+@Composable
+fun OngoingActivityExampleApp(
```

Comment:
Not sure how much you want to fiddle with this class but I think there's a few things that could be improved (some overlap between these options):

1. Handle the user rejecting the POST_NOTIFICATIONS permission. (I guess it should still work, just not display OA.)
2. Switch on Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU instead of creating a nullable `permissionState`, and then switching on that. (Would be nicer if we could either rememberPermissionState() with an empty list of permission *or* construct our own PermissionState object with PermissionState.Granted but neither seems to be possible…)
3. Eliminate the duplicate Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU test. (Perhaps just check if permissionState is null?)
4. Implement/support shouldShowRequestPermissionRationale() (especially since the generic notification permission message doesn't explain that it's required to get the OA indicator).